submodules/externals/speex

```diff
@@ -1 +1 @@
-Subproject commit 6a8d5c8dff15afb04e1c71be3f7e4e051992494b
+Subproject commit 415653bfef10b46d5e7bfa3ad70a68976b77e7b5
```

Comment:
Looking at https://github.com/BelledonneCommunications/linphone-iphone, this is 6a8d5c8dff15afb04e1c71be3f7e4e051992494b, why would I want to use 415653bfef10b46d5e7bfa3ad70a68976b77e7b5?